src/Components/NavBar/NavBar.js

```diff
@@ -1,29 +1,37 @@
-import React, { Component } from "react";
+import React, { useContext } from "react";
 import { NavLink } from "react-router-dom";
+import AuthContext from "../Context/context";
 
 import "./NavBar.css";
 
-class NavBar extends Component {
-	render() {
-		return (
-			<div className="Nav">
-				<h1>JusticeRides</h1>
-				<div style={{ width: "90px" }}>
-					<ul>
+const NavBar = (props) => {
```

Comment:
Your formatting is not standard. Please use 2 tabs and not four across all your files